examples/.gitignore

```diff
@@ -6,3 +6,6 @@ precice-events/
 # PyCharm related files
 .idea
 __pycache__
+
+# Snapshot results
+output/
```

Comment:
If this is folder is created due to a custom test or so, lets not put it in the gitignore. gitignore should only be for ignoring really unavoidable things.

Reply:
The results of a snapshot computation are stored in the output folder. I don't want to upload the hdf5 files related to a specific simulation, which could happen accidentally. The output folder is created automatically. An alternative would be to ignore only the files within the folder.